authentication-service/src/main/kotlin/com/saveourtool/save/authservice/security/ConvertingAuthenticationManager.kt

```diff
@@ -33,7 +33,8 @@ class ConvertingAuthenticationManager(
      */
     override fun authenticate(authentication: Authentication): Mono<Authentication> = if (authentication is UsernamePasswordAuthenticationToken) {
         val (name, identitySource) = authentication.extractUserNameAndIdentitySource()
-        authenticationUserDetailsService.findByUsername(name)
+        val nameAndSource = "$name@SAVE@$identitySource"
```

Comment:
separator should be extracted into single variable into common part

Reply:
Fixed